package.json

```diff
@@ -7,6 +7,10 @@
     "npm": "^8.5.0"
   },
   "license": "MIT",
+  "workspaces": [
+    "docs/examples/*",
```

Comment:
This line makes sure `npm install` and `npm ci` also install webpack's dependencies

Reply:
**quibble** Is that something we absolutely need for the example running? I'd rather we only brought example dependencies when trying to run the example rather than at every install.

Reply:
Thanks @romaricpascal

This was to stop the example(s) going stale or getting out of sync again.

Enables things like:

```console
npm run dev --workspace ./docs/examples/webpack
```

It's possible of course, but then we'd need an extra lockfile (per example) and have to rely on `file://` package paths to the parent `govuk-frontend`. We'd also opt out of:

1. Package updates + auditing
2. Dependabot dependency updates
3. Automatic package deduplication
4. Postinstall `"npm ls --depth=0"` check
5. Example gets a symlink to `govuk-frontend`

I'd recommend the same approach for:

```console
./app/package.json
./tasks/package.json
```

☝️ Not yet of course, but each "area" should manage its own packages and tests

Reply:
Makes sense. Thanks for clarifying 😄 I like the idea of properly separating the app and tasks as well, that sounds pretty neat 😄

Reply:
Yeah 100%

Deployment to Heroku doesn't need all our developer/test tooling etc 😊